### PR TITLE
Update version number to release fix for #2

### PR DESCRIPTION
**Pull request by lsarni**
Forcing the auto-update of the plugin to include the fix to #2 made on #4

### CODE REVIEW

**Comment by lsarni** [comment]
@ignazio1977 we could use auto-update since this plugin is already registered, can you create a new release 4.0.1 here on github? So I can reference the jar [here](https://raw.githubusercontent.com/lsarni/matrix/release-fix/etc/update.matrix.properties) for the `download` property.

I will include the new link on this pull and also the update to `Update-Url:` on the manifest.
That should be enough to get the update published.

File: etc/update.matrix.properties [hunk 1]
```diff
@@ -1,9 +1,8 @@
 
 name=Matrix Views
 id=org.coode.matrix
-version=4.0.0
+version=4.0.1
 download=https://github.com/co-ode-owl-plugins/matrix/releases/download/org.coode.matrix-4.0.0/org.coode.matrix-4.0.0.jar
```
Comment (ignazio1977) [inline etc/update.matrix.properties]: Doesn't this need to be 4.0.1 for the new release to be picked up?

**Review by ignazio1977 — COMMENTED** [review]
The release is available at https://github.com/co-ode-owl-plugins/matrix/releases/download/org.coode.matrix-4.0.1/org.coode.matrix-4.0.1.jar

**Comment by lsarni** [comment]
It worked! Thanks for your help @ignazio1977

**Pull request merged**